### DISCUSSION TITLE
What's the structure of built-in model, b20x256x2?

### DISCUSSION THREAD

**Discussion by HackYardo** [kaorahi/lizgoban#77] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/57677458/162581657-2191ea7e-5d96-49e1-a9ae-78467ff530fd.png)

If is is a b20x256x2 model, why it needs smaller disk space than official model?

**Comment by kaorahi** [comment, marked as answer]
It is not b20 but b15. https://github.com/kaorahi/lizgoban/blob/v0.7.0-pre3/build_with/release.md#to-use-it-on-64bit-windows-immediately

See also:

* (bin.gz & txt.gz) https://github.com/lightvector/KataGo/tree/v1.11.0#where-to-download-stuff
* (getting 15b) https://github.com/lightvector/KataGo/issues/522
